torch/quantization/_quantize_script.py

```diff
@@ -103,13 +104,7 @@ def quantize_script(model, qconfig_dict, run_fn, run_args, inplace=False):
     # revisit after constants is properly handled in
     # JIT
     # torch._C._jit_pass_fold_convbn(model._c)
-    prepare_script(model, scripted_qconfig_dict, True)
+    model = prepare_script(model, scripted_qconfig_dict, True)
```

Comment:
Should the last argument also be `False`?

Reply:
I'll do another refactor to remove this argument